app/services/supplementary-billing/create-billing-invoice.service.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+/**
+ * Creates a billing invoice record
+ *
+ * @module CreateBillingInvoiceService
+ */
+
+const BillingInvoiceModel = require('../../models/water/billing-invoice.model.js')
+const FetchInvoiceAccountService = require('./fetch-invoice-account.service.js')
+
+/**
+ * Create the initial billing invoice for the provided charge version, billing period and billing batch prior to
+ * transactions being sent to the charging module
+ *
+ * @param {Object[]} chargeVersion An Object containing the relevant charge version
+ * @param {Object[]} billingPeriod Object that has a `startDate` and `endDate` that defines the billing period
+ * @param {string} billingBatchId GUID of the billing batch
+ *
+ * @returns {Object} The newly-created billing invoice record
+ */
+async function go (chargeVersion, billingPeriod, billingBatchId) {
+  const billingInvoice = await BillingInvoiceModel.query()
+    .insert({
+      invoiceAccountId: chargeVersion.invoiceAccountId,
+      address: {}, // Have left empty as doesn't appear to be used for SROC
```

Comment:
Now we're committing the code as 'production-ready' we can be more deterministic in our comments.

```suggestion
      address: {}, // Address is set to an empty object for SROC billing invoices
```